src/torrent/data/block.h

```diff
@@ -163,22 +163,16 @@ Block::Block() :
 
 inline BlockTransfer*
 Block::find(const PeerInfo* p) {
-  BlockTransfer* transfer;
-
-  if ((transfer = find_queued(p)) != NULL)
+  if (auto transfer = find_queued(p))
```

Comment:
The style we use is to always check against nullptr.

Reply:
right. this is a C++17 feature.

Would you prefer

```
auto transfer = find_queue(p);
if (transfer != nullptr)
```